### DISCUSSION TITLE
Is it possible to show more pages on ultrawide monitor?

### DISCUSSION THREAD

**Discussion by zhizhang77** [sumatrapdfreader/sumatrapdf#3269] [Ideas]
An ultrawide monitor has enough space to show 3-4 pages simultaneously, but currently we can only select "single page view" or "facing view" so a lot of area get wasted. Is it possible to add options for 3-pages or 4-pages view? Thanks.

**Comment by GitHubRulesOK** [comment]
hmm generally no but see  https://github.com/sumatrapdfreader/sumatrapdf/issues/2952 and https://github.com/sumatrapdfreader/sumatrapdf/issues/246#issuecomment-798529197
